packages/components/package.json

```diff
@@ -23,7 +23,9 @@
         "typecheck": "tsc --noEmit",
         "test": "vitest run --coverage",
         "test:ui": "vitest --ui",
-        "test:watch": "vitest"
+        "test:watch": "vitest",
+        "test:integ": "cypress open --component",
+        "test:integ:ci": "cypress run --component --headless --browser electron"
```

Comment:
for now is only electron, but we can (_should?_) expand to Chrome too in parallel